src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -14,7 +14,6 @@ import {
 } from "@stacks/transactions";
 import { serverBroadcastTx } from "@/actions/server-broadcast-tx";
 import {
-  callContractAsigna,
```

Comment:
nit: do we still need `callContractAsigna` or the fn itself can be removed as well?

Reply:
fair